internal/cli/up.go

```diff
@@ -53,6 +55,14 @@ func (c *upCmd) run(ctx context.Context, args []string) error {
 
 	logOutput(fmt.Sprintf("Starting Tilt (built %s)…\n", buildDateStamp()))
 
+	if trace {
```

Comment:
maybe put this in cli.go since it's not `up` specific and if there are ever other tilt commands we'll probs want this logic too

Reply:
Yeah, that was where I first tried to put it, but we can't get the trace id there since we haven't made a span yet, and I wasn't sure I wanted to make a span just for that purpose. I figured that's a thing we can worry about if/when we have other commands (there's probably already other stuff there we'll want to generalize anyway).